provider/virtio_fs_utils.py

```diff
@@ -53,9 +51,14 @@ def basic_io_test(test, params, session):
     host_data = os.path.join(fs_source, test_file)
     try:
         if windows:
+            cmd_dd = params.get("virtio_fs_cmd_dd",
+                                'dd if=/dev/random of=%s bs=1M count=100')
             driver_letter = get_virtiofs_driver_letter(test, fs_target, session)
             fs_dest = "%s:" % driver_letter
         else:
+            cmd_dd = params.get("virtio_fs_cmd_dd",
+                                'dd if=/dev/urandom of=%s bs=1M '
+                                'count=100 iflag=fullblock')
```

Comment:
I am confused with this part, according to the commit message, just to adapt the different `cmd_dd` to linux and windows OS, I think we could handle it from the cfg file:
```
windows:
    virtio_fs_cmd_dd = xxxxx
linux:
    virtio_fs_cmd_dd = yyyyy
```

Reply:
As it may be not that possible for the dd cmd to be changed, we supply a default basic dd cmd in py to avoid too many same settings in cfg files, cc @YongxueHong @nickzhq

Reply:
> I am confused with this part, according to the commit message, just to adapt the different `cmd_dd` to linux and windows OS, I think we could handle it from the cfg file:
> 
> ```
> windows:
>     virtio_fs_cmd_dd = xxxxx
> linux:
>     virtio_fs_cmd_dd = yyyyy
> ```

Hello @YongxueHong , the virtio_fs_utils.py was designed to the public module, those cmd_dd should be prepared in the file as the default value. In addition, this way to allow the users to provider params in cfg.

cc @zhencliu